src/utils/index.js

```diff
@@ -54,3 +54,61 @@ export function convertFormDataToObject( formData, includedFields = [] ) {
 		return acc;
 	}, {} );
 }
+
+/**
+ * Register a reader activity dispatch on an element event.
+ *
+ * @param {string|Element} element The element to register the activity on. Can either be the element node or a string for the element selector.
+ * @param {string}         action  The action to dispatch an activity for.
+ * @param {Function}       cb      The callback to populate the activity data.
+ * @param {string}         event   The element event to listen for. Defaults to `submit` for form elements and `click` for other elements.
+ */
+export function registerElementActivity( element, action, cb, event ) {
+	window.newspackRAS = window.newspackRAS || [];
+	if ( typeof element === 'string' ) {
+		element = document.querySelector( element );
```

Comment:
I think this needs to be `document.querySelectorAll` so that the function can handle multiple instances of the same selector on a page. Right now the `payment_method_deleted` event will only fire when deleting the first payment method in the table, as each payment method in the table has its own "Delete" button.

Reply:
Ah, great catch! Fixed in 69df8db8c62e45c859fe5c08c678a85d778e5073

Reply:
Confirmed!